src/components/Header/Header.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { FC } from 'react';
-import { Group, AppShell } from '@mantine/core';
+import { Group, AppShell, Burger } from '@mantine/core';
+import { useNavBarToggler } from '~/hooks/useNavbarToggler';
+
+export const Header: FC<React.PropsWithChildren > = ({ children }) => {
+    const [isNavbarVisible, {toggle}] = useNavBarToggler();
 
-export const Header: FC<React.PropsWithChildren> = ({ children }) => {
     return (
         <AppShell.Header>
             <Group h="100%" px="md" justify="space-between">
+                <Burger opened={isNavbarVisible} onClick={toggle} hiddenFrom="xs" size="sm" />
```

Comment:
```suggestion
                <Burger opened={isNavbarVisible} onClick={toggle} hiddenFrom="xs" size="xs" />
```

Размер лучше установить в `xs` для визуальной совместимости с остальными элементами.

Reply:
добавил фикс